### DISCUSSION TITLE
Ignore word seperation in URL when hovering on tabs

### DISCUSSION THREAD

**Discussion by MaxHasBeenUsed** [mbnuqw/sidebery#1805] [Q&A]
When a URL is too long to display in one line, it leaves awkward spaces at end of each line. Take below, a Google map link for an example.

Ideally I want it to create a new line once it reaches end of line whether there is a whole word in link or not. Because in most cases, these long long URLs contains lots of random strings that makes no sense to human brain. Take below, a Google map link for an example.

Wondering if some css in style settings can do the trick? Also, feel free to convert this to an issue if necessary.

![图片](https://github.com/user-attachments/assets/6b2d6d4b-9197-4aee-ac13-d9ee6f71aa04)

**Comment by ChilledMoth** [comment]
Those are native browser tooltips. Styling (including applying different line/word breaking rules) would have to be done outside of the extension in a custom userChrome.css file. However, this can get messy real quick as selectors would have to be very specific in order not to interfere with other tooltips.

Try setting 'Show URL in tooltip' to 'stripped' in the tabs section of the settings page as it automatically truncates the URL by removing all URL parameters and thus gets rid of most of the 'random' strings.

**Comment by MaxHasBeenUsed** [comment]
> However, this can get messy real quick as selectors would have to be very specific in order not to interfere with other tooltips.

Ah I see, I've just noticed the same line/word breaking behavior in website titles. So guessing as long as Sidebery uses native browser tooltip, there's no good way to apply different style settings to titles and URLs?
